src/intersection-observer-polyfill.js

```diff
@@ -33,6 +34,133 @@ import {layoutRectLtwh, rectIntersection, moveLayoutRect} from './layout-rect';
  */
 export let DOMRect;
 
+export const AMP_DEFAULT_THRESHOLD =
```

Comment:
I don't have a strong preference so I removed it. My point was `DEFAULT_THRESHOLD` is [0], while `AMP_DEFAULT_THRESHOLD` is [0, 0.05...1]